project.py

```diff
@@ -28,6 +27,11 @@ def __init__(self):
         self.width = None
         self.height = None
         self.saved_cnt = None
+        self.seed = None
+        self.numberOfSlides = None
```

Comment:
@Foxiy I do not want to nit pick, but this smells like Java =)

[PEP 8](https://www.python.org/dev/peps/pep-0008/) is directly against that.